indico/modules/categories/controllers/display.py

```diff
@@ -330,3 +332,23 @@ def _process(self):
                                       session.user,
                                       Event.end_dt >= now_utc())
         return send_file(filename, buf, 'application/atom+xml')
+
+
+class RHCategoryCalendarView(RHDisplayCategoryBase):
+    def _process(self):
+        form = CategoryCalendarForm(timezone=self.category.timezone)
+        serialized_events = {}
+        if form.validate_on_submit():
+            category_events = get_category_timetable([self.category.id], form.from_dt.data, form.to_dt.data)['events']
+            tz = self.category.display_tzinfo
+            for day, event_data in category_events.iteritems():
+                for dt, ev in event_data:
+                    if ev.id in serialized_events:
+                        continue
+                    serialized_events[ev.id] = {'title': ev.title, 'start': format_datetime(ev.start_dt, timezone=tz),
+                                                'end': format_datetime(ev.end_dt, timezone=tz),
+                                                'url': url_for('event.conferenceDisplay', ev)}
+        return WPCategoryStatistics.render_template('display/calendar.html',
```

Comment:
`WPCategory` - this has nothing to do with statistics :)

        return WPCategory.render_template('display/calendar.html', self.category, form=form,
                                          serialized_events=serialized_events.values())